composer.json

```diff
@@ -20,10 +20,12 @@
     "require": {
         "php": ">=5.5.9",
         "robmorgan/phinx": "0.6.6",
-        "cakephp/cakephp": "~3.2"
+        "cakephp/orm": "~3.2",
+        "cakephp/cache": "~3.2"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.1",
+        "cakephp/cakephp": "~3.2",
```

Comment:
@dereuromark probably I didn't understand what you mean. Can you tell me what split are you telling and what is necessary ?